README.md

```diff
@@ -322,7 +349,7 @@ class App extends Component {
     this.setState({error: true, errorMsg}) 
   }
 
-  // This is a great chance save data internally
+  // This is a great chance save data internally if needed 
```

Comment:
Maybe have the comment read:
// If you'd like to save the data on your own platform in a PII-safe way, use the response object.

Also, is the response object documented?

Reply:
Comment changed. 

It is documented above that it is an Account object but I will rename the var to reflect it as well. Response is definitely a bit too generic